adapter/parser/src/main/java/org/mobilitydata/gtfsvalidator/parser/GtfsEntityParser.java

```diff
@@ -185,6 +185,29 @@ public ParsedEntity parse(RawEntity toParse) {
                                 ));
                     }
 
+                /*
```

Comment:
I think this code works here, however in `GtfsTypeValidator` there is a fragment of code (related to `Time` validation) to remove to avoid duplication. Variables should be `final`, some of them could be renamed for legibility

```suggestion
                /*
                } else if (columnSpecProto.getType().getType() ==
                        GtfsSpecificationProto.ColumnInputType.InputType.TIME) {
                    //Shouldn't we implement this elsewhere? Custom validator or class?
                    Integer timeValue = null;
                    if (rawField != null && rawField.matches("([0-9][0-9]|[0-9]):[0-5][0-9]:[0-5][0-9]")) {
                        final String[] timeStringSplit = rawField.split(":");
                        final String hourString = timeStringSplit[0];
                        final String minuteString = timeStringSplit[1];
                        final String secondString = timeStringSplit[2];
                        final Integer hourValue = Integer.valueOf(hourString);
                        final Integer minuteValue = Integer.valueOf(minuteString);
                        final Integer secondValue = Integer.valueOf(secondString);
                        //Converting time to an Integer value representing number of seconds
                        final timeValueAsInt = hourValue * 3600 + minuteValue * 60 + secondValue;
                        //Setting noon as point zero, subtracting 12 * 3600 seconds
                        final timeValueAsIntFromNoonOfDayOfService = timeValue - 12 * 3600;
                    }
                    contentByHeaderMap.put(columnSpecProto.getName(), timeValue);
                    
```

Reply:
Also, this implementation requires additional tests

Reply:
I modified it, but kept as commented code for now as we should discuss removing `Time` validation from `GtfsTypeValidator` (it implies removing lots of code related to the time validation)

Reply:
To be implemented in TimeConversionUtils class